Code/API/Python/tests/test_vecteur.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import annotations
+import unittest
+from funibot_api.funibot import Vecteur
+
+class TestVecteur(unittest.TestCase):
+
+    def test_repr(self):
+        """Test d'un vecteur pour voir s'il se représente bien"""
+        VecteurTest = Vecteur(3,-2,7)
```

Comment:
Les noms de variables en Python suivent la convention du "snake_case", c'est à dire tout en minuscules, avec des underscores entre les mots.

Ce sont les noms de classes qui sont en PascalCase comme tu as fait.

```suggestion
        vecteur_test = Vecteur(3,-2,7)
```